### PR TITLE
PR: Update Spanish translation for Spyder 4

### PR DESCRIPTION
**Pull request by juanis2112**
This is part of issue #10672.

### CODE REVIEW

File: spyder/locale/es/LC_MESSAGES/spyder.po [hunk 1]
```diff
@@ -1422,19 +1427,21 @@ msgstr "Seleccionar una nueva fuente"
 
 #: spyder/plugins/completion/languageserver/confpage.py:179
 msgid "TCP port number of the server"
-msgstr ""
+msgstr "Número de puerto TCP del servidor"
 
 #: spyder/plugins/completion/languageserver/confpage.py:189
 msgid "Additional arguments required to start the server"
-msgstr ""
+msgstr "Argumentos adicionales son necesarios para iniciar el servidor"
```
Comment (andfoy) [inline spyder/locale/es/LC_MESSAGES/spyder.po]: I think this one should be "Argumentos adicionales necesarios para iniciar el servidor"

File: spyder/locale/es/LC_MESSAGES/spyder.po [hunk 2]
```diff
@@ -1468,29 +1475,33 @@ msgstr "Opciones avanzadas"
...
 #: spyder/plugins/completion/languageserver/confpage.py:329
 msgid "Command used to start the LSP server locally. Must be non empty"
 msgstr ""
+"Comando utilizado para iniciar el servidor LSP localmente. No debe estar "
+"vacío"
 
 #: spyder/plugins/completion/languageserver/confpage.py:336
 msgid "Program was not found on your system"
-msgstr ""
+msgstr "El programa no se encontró en su sistema"
```
Comment (andfoy) [inline spyder/locale/es/LC_MESSAGES/spyder.po]: "No se encontró el programa en el sistema"?

File: spyder/locale/es/LC_MESSAGES/spyder.po [hunk 3]
```diff
@@ -1532,7 +1544,7 @@ msgstr "Mostrar contenidos de listas"
 
 #: spyder/plugins/completion/languageserver/confpage.py:723
 msgid "Show automatic completions after characters entered:"
-msgstr ""
+msgstr "Mostrar completado automático después de ingresar los caracteres:"
```
Comment (andfoy) [inline spyder/locale/es/LC_MESSAGES/spyder.po]: "Número de caractéres mínimos para mostrar completado automáticamente?"

File: spyder/locale/es/LC_MESSAGES/spyder.po (hunk 1 above)
Comment (juanis2112) [reply spyder/locale/es/LC_MESSAGES/spyder.po]: Right thanks! I'll fix it in the next commit.

File: spyder/locale/es/LC_MESSAGES/spyder.po (hunk 2 above)
Comment (juanis2112) [reply spyder/locale/es/LC_MESSAGES/spyder.po]: Much better, I'll fix it in the next commit.

File: spyder/locale/es/LC_MESSAGES/spyder.po (hunk 3 above)
Comment (juanis2112) [reply spyder/locale/es/LC_MESSAGES/spyder.po]: Agree. I'll fix it in the next commit.

File: spyder/locale/es/LC_MESSAGES/spyder.po [hunk 4]
```diff
@@ -241,9 +240,8 @@ msgstr ""
 "para todas las sesiones)"
 
 #: spyder/app/mainwindow.py:810
-#, fuzzy
 msgid "Install Kite completion engine"
-msgstr "Usar un widget para completar texto"
+msgstr "Intalar motor de completado de Kite"
```
Comment (andfoy) [inline spyder/locale/es/LC_MESSAGES/spyder.po]: *Instalar

Comment (ccordoba12) [reply spyder/locale/es/LC_MESSAGES/spyder.po]: Thanks @andfoy. I'm reviewing @juanis2112 work right now and fixing errors like this one.

**Pull request merged**